### PR TITLE
Improvements while going through the Elixir guide

### PR DESCRIPTION
**Pull request by GregMefford**
While working through the guide, I found a few things to correct grammatically, and also tried to clarify a few things that I think may not be obvious to new-comers.

### CODE REVIEW

**Comment by GregMefford** [comment]
It looks like `CODEOWNERS` is set up to be global to the whole repo, so just a heads up to @open-telemetry/erlang-approvers about this as well.

**Review by ferd — APPROVED** [review]

**Review by flands — APPROVED** [review]

**Pull request merged**
